apps/finance/app/src/components/Transfers.js

```diff
@@ -173,19 +176,23 @@ class Transfers extends React.Component {
           </NoTransfers>
         ) : (
           <div>
-            <GetWindowSize>
-              {({ width }) => (
+            <Viewport>
+              {({ width, breakpoints }) => (
                 <FixedTable
                   header={
-                    <BreakPoint from="medium">
-                      <TableRow>
-                        <DateHeader title="Date" />
-                        <SourceRecipientHeader title="Source / Recipient" />
-                        <ReferenceHeader title="Reference" />
-                        <AmountHeader title="Amount" align="right" />
-                        <TableHeader />
-                      </TableRow>
-                    </BreakPoint>
+                    <Viewport>
```

Comment:
Do we need this extra `Viewport` when we already have an instance above?

Reply:
🤦‍♂️ thanks!